draft-ietf-acme-acme.md

```diff
@@ -1428,6 +1436,35 @@ over the same type, e.g., ["http-03", "http-05"].  Challenges within a type are
 testing the same capability of the domain owner, and it may not be possible to
 satisfy both at once. ]]
 
+## Validation of Wildcard Authorizations
```

Comment:
So you want `wildcard-http-01`, `wildcard-tls-sni-01` and `wildcard-dns-01`? That seems excessive. Adding `"wildcard":true` to the authorization object seems preferable.